master/buildbot/statistics/capture.py

```diff
@@ -53,16 +54,15 @@ def _store(self, post_data, series_name, context):
                                         context)
 
 
-class CaptureProperty(Capture):
+class CapturePropertyBase(Capture):
 
     """
-    Convenience wrapper for getting statistics for filtering.
-    Filters out build properties specifies in the config file.
+    A base class for CaptureProperty* classes.
     """
 
-    def __init__(self, builder_name, property_name, callback=None):
-        self._builder_name = builder_name
+    def __init__(self, property_name, callback=None, regex=False):
         self._property_name = property_name
+        self._regex = regex
```

Comment:
should some sanity check be performed here?  like it's either a string or an instance of compiled regex?